### PR TITLE
osd: minor performance improvements

### PR DESCRIPTION
**Pull request by xiexingguo**

### CODE REVIEW

**Comment by xiexingguo** [comment]
@athanatos Your comments are appreciated.

File: src/osd/OSD.cc [hunk 1]
```diff
@@ -4124,7 +4124,7 @@ void OSD::heartbeat_entry()
   while (!heartbeat_stop) {
     heartbeat();
 
-    double wait = .5 + ((float)(rand() % 10)/10.0) * (float)cct->_conf->osd_heartbeat_interval;
+    double wait = .5 + (double)(rand() / (double)RAND_MAX) * (double)cct->_conf->osd_heartbeat_interval;
```
Comment (athanatos) [inline src/osd/OSD.cc]: Have you actually observed this being a problem?  I'm not convinced that this is a meaningful change.

**Comment by xiexingguo** [comment]
@athanatos Dropped that change, thanks!

**Comment by yuriw** [comment]
re-run
http://pulpito.ceph.com/yuriw-2016-08-03_08:24:51-rados-wip-yuri-testing2_2016_08_01-distro-basic-smithi/

http://pulpito.ceph.com/yuriw-2016-08-01_17:57:24-rados-wip-yuri-testing2_2016_08_01-distro-basic-smithi/

**Pull request merged**